amy/communityroles/fields.py

```diff
@@ -17,7 +17,10 @@ def apply_labels(self, labels: list[str]) -> None:
 
     def get_context(self, name: str, value: str, attrs: dict):
         value_deserialized = json.loads(value)
-        value_deserialized_dict = dict(value_deserialized)
+        if value_deserialized is not None:
+            value_deserialized_dict = dict(value_deserialized)
+        else:
+            value_deserialized_dict = {}
```

Comment:
`json.loads` can potentially return other types that will fail to dict transformation, for example:

```python
>>> S = '["asd", "dsa", "zxc"]'
>>> json.loads(S)
['asd', 'dsa', 'zxc']
>>> dict(json.loads(S))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: dictionary update sequence element #0 has length 3; 2 is required
```

I'm wondering if `try-except` wouldn't be better here. You can add logging of the failed value, too, so in any case we could diagnose faster.